newIDE/app/src/UI/Theme/SolarizedDarkTheme/EventsSheet.css

```diff
@@ -0,0 +1,218 @@
+/**
+ * This is the styling applied to an EventsSheet.
+ * This also overwrite the default react-sortable-tree css.
+ */
+
+ .gd-events-sheet-nord-theme.events-tree {
+  /* Use the fonts provided by the operating system(s) as possible. */
+  /* If you update this font list, be sure to do it in all the other places using fonts in the codebase. */
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+    Oxygen-Sans, Ubuntu, 'Helvetica Neue', Helvetica, sans-serif,
+    'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol' !important;
+  font-size: 14px !important;
+  background-color: #224ea7;
+}
+
+/**
+ * Ensure scroll is only on Y direction, preventing the display
+ * of horizontal scrollbar.
+ */
+.gd-events-sheet-nord-theme .rst__virtualScrollOverride {
```

Comment:
You'll need to rename everything called xxx-nord-theme to xxx-solarized-dark-theme in all CSS files and update the JavaScript file to set this style name :)